### ISSUE TITLE
ResourceWatcher is not working since 1.7.0

### ISSUE THREAD

**Issue by redking** [alexo/wro4j#129]
Hey,

My wro configuration has a non-zero value for resourceWatcherUpdatePeriod and, in previous versions, a modification to a Less or JS file that was referenced in a wro.xml package would trigger a rebuild. 

Since updating to wro4j 1.7.0, this no longer happens, in the following cases
1. If a modification is made to a JavaScript file that is directly referenced in a wro.xml package, or that is referenced in another package, included via a group-ref tag
2. If a modification is made to a Less or Css file that is included via a group-ref tag

So, in summary, the cache is _not_ refreshed if I modify a JavaScript file, or any file in a package imported via group-ref. I raised a similar issue a few versions ago https://code.google.com/p/wro4j/issues/detail?id=581

If I revert back to wro 1.6.2 (the last version we could use), the issue no longer occurs. Unfortunately, we have to use wro 1.7.0 as we need to use Less 1.4.0. 

This is a pretty critical problem as it slows down development significantly.

(Testing was on Windows 7, incidentally)
